### ISSUE TITLE
Issue installing on MacOS with M2

### ISSUE THREAD

**Issue by speezlab** [ianberdin/xlsx-to-csv#1]
Hi - issuing the following command `npm install --save xlsx-to-csv` results in the following error:
```
npm ERR! command failed
npm ERR! command sh /var/folders/xh/jty51n617s5053ycj_fpqdb80000gn/T/postinstall-57f5c9bf.sh
npm ERR! Downloading release from https://github.com/ianberdin/xlsx-to-csv/releases/download/v1.0.1/xlsx-to-csv-aarch64.tar.gz
npm ERR! Error fetching release: Request failed with status code 404
```

Note it's downloading the release from `https://github.com/ianberdin/xlsx-to-csv/releases/download/v1.0.1/xlsx-to-csv-aarch64.tar.gz`. I checked your releases and believe it's looking for this: `https://github.com/ianberdin/xlsx-to-csv/releases/download/v1.0.1/xlsx-to-csv-macos-aarch64.tar.gz` (note the missing `..-macos-aarch64..`)

Please advise.   Thank you!

**Comment by iifawzi** [comment]
I have the same issue, were you able to fix it @speezlab?
using MacOS with M1 MAX

**Comment by 70747H** [comment]
@ianberdin  I have the same issue, were you able to fix it @speezlab ?
using MacOS with M1 Pro.

**Comment by ianberdin** [comment]
Hi guys.
I'm, actually, forgot about this package. Do you really need it?

**Comment by 70747H** [comment]
Yes, i used it for an app which is in production now 
But cannot compile it on my local machine, the M1

**Comment by ianberdin** [comment]
Published a fix [1.0.2](https://www.npmjs.com/package/xlsx-to-csv/v/1.0.2) version.
Thank you all.

**Comment by 70747H** [comment]
@ianberdin still facing the same old issue, do you have any idea what's wrong?

![Screenshot 2023-06-05 at 8 22 06 AM](https://github.com/ianberdin/xlsx-to-csv/assets/12988477/2511acd0-fcbc-46a2-85d2-ceda97bc4d87)
